src/test/java/teammates/test/cases/ui/browsertests/AutomatedSessionRemindersTest.java

```diff
@@ -28,6 +29,20 @@ public static void classSetup() throws Exception {
         printTestClassHeader();
         testData = loadDataBundle("/AutomatedSessionRemindersTest.json");
         
+        /* 
+         * In this test, we set the email address of the accounts to be the same as the
+         * support email address. When running the test against a production server,
+         * email alerts will be sent to the specified support email address.
+         * The tester should manually check the email box after running the test suite. 
+         */
+        
+        testData.accounts.get("instructorWithEvals").email = Config.SUPPORT_EMAIL;
+        testData.instructors.get("AutSessRem.instructor").email = Config.SUPPORT_EMAIL;
+        testData.students.get("alice.tmms@AutSessRem.course").email = Config.SUPPORT_EMAIL;
+        testData.feedbackSessions.get("closingSession").creatorEmail = Config.SUPPORT_EMAIL;
+        testData.feedbackSessions.get("openingSession").creatorEmail = Config.SUPPORT_EMAIL;
+        testData.feedbackSessions.get("publishedSession").creatorEmail = Config.SUPPORT_EMAIL;
+        testData.feedbackQuestions.get("question").creatorEmail = Config.SUPPORT_EMAIL;
```

Comment:
Dr @damithc I can't say that this is an elegant solution, but this is the only way I can think of that both achieves what I want to achieve and also doesn't make your life harder.